### PR TITLE
Feature/asb 310 endpoint to fetch a single representation

### PR DESCRIPTION
**Pull request by lvnhmd**

### CODE REVIEW

**Review by cpcundill — CHANGES_REQUESTED** [review]
Needs a catchup with `main` branch

File: packages/applications-service-api/__tests__/unit/controllers/representations.test.js [hunk 1]
```diff
@@ -70,42 +78,37 @@ const returnData = {
...
-});
-
 jest.mock('../../../src/lib/config.js', () => ({
   logger: {
     level: process.env.LOGGER_LEVEL || 'info',
   },
   itemsPerPage: 3,
 }));
 
+jest.mock('../../../src/services/representation.service');
```
Comment (cpcundill) [inline packages/applications-service-api/__tests__/unit/controllers/representations.test.js:91]: I appreciate it is much easier to mock the servicer due to the complexity of mocking Sequelize.  However, we are now missing a test coverage for the representation business service.  Happy for that to be a separate unit test file, but I think we shouldn't avoid it due to the difficulty.  Let's just find a solution together to manage the complexity.

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/applications-service-api/src/services/representation.service.js [hunk 2]
```diff
@@ -43,6 +47,27 @@ const getRepresentationsForApplication = async (applicationId, page, searchTerm)
   return representations;
 };
 
+const getRepresentationById = async (ID) => {
+  return db.Representation.findOne({ where: { ID } });
+};
+const getFilters = async (filter, applicationId) => {
```
Comment (cpcundill) [inline packages/applications-service-api/src/services/representation.service.js:53]: This shouldn't be needed right?

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/applications-service-api/__tests__/unit/controllers/representations.test.js (hunk 1 above)
Comment (cpcundill) [reply packages/applications-service-api/__tests__/unit/controllers/representations.test.js:91]: What do you think @lvnhmd ?

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/applications-service-api/__tests__/unit/controllers/representations.test.js (hunk 1 above)
Comment (lvnhmd) [reply packages/applications-service-api/__tests__/unit/controllers/representations.test.js:91]: @cpcundill - I am working on it. Opted for a separate unit test file.

**Review by cpcundill — APPROVED** [review]

**Pull request merged**